playground/playground.js

```diff
@@ -1,6 +1,12 @@
+const newProjectStr = 'createEmptyProject';
```

Comment:
For compatibility with older browsers, I don't think we can support `const`. how about `var NEW_PROJECT_HASH'?